### ISSUE TITLE
Solution properties are not exported to JSON

### ISSUE THREAD

**Issue by scaverod** [mork-optimization/mork#229]
To add the properties to the JSON I have developed the following code:
    
```
    /**
     * Get all custom properties
     * @return a map with the property names as keys,
     * and the property value as value
     */
    public Map<String, Object> getCustomProperties() {
        var properties = new HashMap<String, Object>();
        for (var entry : customProperties().entrySet()) {
            properties.put(entry.getKey(), entry.getValue().apply(this));
        }
        return properties;
    }
```

BTW; IDK what does lastModifiedTime mean. is it smth. a human should understand 😅?

**Comment by rmartinsanta** [comment]
Will be fixed in v0.21, will be released soon.

`lastModifiedTime` represents the number of ellapsed nanoseconds since the current WorkUnit started being processed. It is internally used by the framework to calculate the Time To Best (TTB) metric. 

This can in turn help users during the experimentation, by detecting and reporting if the current solution has not being modified for a long time (or in other words, you are trying to hard to modify a solution that may already be optimal).

**Comment by rmartinsanta** [comment]
Fixed by a706ef3d617cc12020f192d21769632e699dcef1
